src/main/java/org/jenkinsci/test/acceptance/junit/Native.java

```diff
@@ -45,9 +63,34 @@ public void evaluate() throws Throwable {
 
                 private void verifyNativeCommandPresent(Native n) throws IOException, InterruptedException {
                     if (n==null)        return;
+                    
+                    // Checks performed in this annotation only makes sense if Jenkins and Tests are being executed in the same machine
+                    if (!(controller instanceof LocalController)) {
+                        throw new AssumptionViolatedException("Test skipped. Native should be used with a local controller, otherwise it cannot be fully trusted to work as expected.");
```

Comment:
I'm not sure that because you don't know something is true that you should treat it as false here in this case.

We do not run the tests like this in the CI system - so this would mainly hit local users, and I or others may want to run the tests against a `RemoteJenkinsController` so I can run Unix based tests.  Now I have no way to do that, before it could do this by virtue of creating a dummy file in my path, if it had not picked up something from cygwin to begin with...